Source/SwiftLintFramework/Rules/LineLengthRule.swift

```diff
@@ -31,15 +31,37 @@ public struct LineLengthRule: ConfigurationProviderRule, SourceKitFreeRule {
     )
 
     public func validate(file: File) -> [StyleViolation] {
-        let minValue = configuration.params.map({ $0.value }).min(by: <)
+        let minValue = configuration.params.map({ $0.value }).min(by: <) ?? Int.max
```

Comment:
I know this was already this way, but you can just use `min()` instead of `min(by: <)`